HackingWithSwift/week2/day_7/Closures with multiple parameters/Closures with multiple parameters .playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+func coocking(Ingredient: (String, Int, String, Int) -> String) {
```

Comment:
Arguments names should be lowercased.

Reply:
You can also name your closure arguments to make them more readable for other programmers who will be working with your code. 

Please note that you have misspelled the word “cooking”